src/OpenFeature/Api.cs

```diff
@@ -218,6 +219,59 @@ public EvaluationContext GetContext()
             }
         }
 
+        /// <summary>
+        /// Return the transaction context propagator.
+        /// </summary>
+        /// <returns><see cref="ITransactionContextPropagator"/>the registered transaction context propagator</returns>
+        public ITransactionContextPropagator GetTransactionContextPropagator()

```

Comment:
I don't think we need this accessor, if we need it for tests that's fine, but let's make it internal in that case.

Reply:
I followed the [java-sdk](https://github.com/open-feature/java-sdk/blob/f1817d8fef585f957de1cfb9222b03cb591ed2e9/src/main/java/dev/openfeature/sdk/OpenFeatureAPI.java#L139) implementation. I am happy to change it to `internal` if it is the expected behaviour.

Let me know if I should remove or mark it as internal.

Reply:
Ah, I didn't realize we implemented it there. I'm not of the opinion it's a great idea, but since it's also there it's not a blocker for me (it's not in the spec).

I can go either way... do you think we should have it? 

Approving now.

Reply:
Does anyone outside of the client **need** to access the Transaction Context Propagator? Likely not... 

I prefer to keep it `internal` so we expose what is necessary. But I don't like different language implementations going in different directions, and I don't know what the best approach is.

Reply:
> I prefer to keep it internal so we expose what is necessary.

I agree.

> But I don't like different language implementations going in different directions, and I don't know what the best approach is.

Of course I generally agree, that's why we have a spec, but there's always going to be some deviations and extras. If anything, we should probably deprecate the Java function and remove it eventually. I don't see it in JS either.

Reply:
Change visibility here: https://github.com/open-feature/dotnet-sdk/pull/312/commits/bf4dbb4ff1d0a399aeb9e4d424be755d9dc05920